tests/testthat/test-printing.R

```diff
@@ -298,16 +298,18 @@ test_that("Inset works for table, ref_footnotes, and main footer", {
 
   # Building the table and trimming NAs
   tt <- build_table(lyt, DM)
-  tt <- trim_rows(tt)
+  tt <- prune_table(tt)
```

Comment:
adding prunning is good, but keep the trim_rows as well.

If they result in identical tables in this case, make that an additional test.

Reply:
I will add additional tests to test it as it at the moment but prune and trim produce different tables